<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testsuite failures="1" name="cucumber.runtime.formatter.JUnitFormatter" skipped="0" tests="1" time="38.763177">
<testcase classname="Cleartrip Application Login" name="Book ticket in Cleartrip" time="38.763177">
<failure message="cucumber.runtime.CucumberException: Arity mismatch: Step Definition 'com.CucumberCraft.stepDefinitions.CleartripStepDef.user_selects_both_depart_and_return(String,String) in file:/C:/CRAFT/CRAFT_Cucumber/target/test-classes/' with pattern [^User selects both Depart and return$] is declared with 2 parameters. However, the gherkin step has 0 arguments []. &#10;Step: Then User selects both Depart and return&#13;&#10;&#9;at cucumber.runtime.StepDefinitionMatch.arityMismatch(StepDefinitionMatch.java:102)&#13;&#10;&#9;at cucumber.runtime.StepDefinitionMatch.transformedArgs(StepDefinitionMatch.java:60)&#13;&#10;&#9;at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)&#13;&#10;&#9;at cucumber.runtime.Runtime.runStep(Runtime.java:300)&#13;&#10;&#9;at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)&#13;&#10;&#9;at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)&#13;&#10;&#9;at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:44)&#13;&#10;&#9;at cucumber.runtime.model.CucumberScenarioOutline.run(CucumberScenarioOutline.java:46)&#13;&#10;&#9;at cucumber.runtime.model.CucumberFeature.run(CucumberFeature.java:165)&#13;&#10;&#9;at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:63)&#13;&#10;&#9;at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)&#13;&#10;&#9;at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;&#10;&#9;at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;&#10;&#9;at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;&#10;&#9;at java.lang.reflect.Method.invoke(Method.java:498)&#13;&#10;&#9;at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)&#13;&#10;&#9;at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)&#13;&#10;&#9;at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)&#13;&#10;&#9;at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)&#13;&#10;&#9;at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)&#13;&#10;&#9;at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;&#10;&#9;at org.testng.TestRunner.privateRun(TestRunner.java:744)&#13;&#10;&#9;at org.testng.TestRunner.run(TestRunner.java:602)&#13;&#10;&#9;at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)&#13;&#10;&#9;at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)&#13;&#10;&#9;at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)&#13;&#10;&#9;at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)&#13;&#10;&#9;at java.util.concurrent.FutureTask.run(FutureTask.java:266)&#13;&#10;&#9;at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)&#13;&#10;&#9;at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)&#13;&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#13;&#10;"><![CDATA[Given User is on cleartrip landing page.....................................passed
When User selects the "Roundtrip" options...................................passed
When User enters destinations from kolkata and To chennai...................passed
Then User selects both Depart and return....................................failed
And User enters Passenger Details...........................................skipped
Then User decides class and Airlines........................................skipped
And User click on Search Flight button......................................skipped

StackTrace:
cucumber.runtime.CucumberException: Arity mismatch: Step Definition 'com.CucumberCraft.stepDefinitions.CleartripStepDef.user_selects_both_depart_and_return(String,String) in file:/C:/CRAFT/CRAFT_Cucumber/target/test-classes/' with pattern [^User selects both Depart and return$] is declared with 2 parameters. However, the gherkin step has 0 arguments []. 
Step: Then User selects both Depart and return
	at cucumber.runtime.StepDefinitionMatch.arityMismatch(StepDefinitionMatch.java:102)
	at cucumber.runtime.StepDefinitionMatch.transformedArgs(StepDefinitionMatch.java:60)
	at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)
	at cucumber.runtime.Runtime.runStep(Runtime.java:300)
	at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)
	at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)
	at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:44)
	at cucumber.runtime.model.CucumberScenarioOutline.run(CucumberScenarioOutline.java:46)
	at cucumber.runtime.model.CucumberFeature.run(CucumberFeature.java:165)
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:63)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]></failure>
</testcase>
</testsuite>
